libs/ui/src/print_element.tsx

```diff
@@ -1,56 +1,17 @@
 import React, { useEffect } from 'react';
 import ReactDom from 'react-dom';
-import styled from 'styled-components';
 
 import { ElementWithCallback, PrintOptions } from '@votingworks/types';
 import { assert, getPrinter } from '@votingworks/utils';
 
-const ScreenDisplayNone = styled.div`
-  display: none;
-  @media print {
-    display: block;
-  }
-`;
-
-interface PrintElementBaseOptions {
-  screenDisplayNone?: boolean;
-}
-
-export type PrintElementToPdfOptions = PrintElementBaseOptions;
-
-export interface PrintElementOptions
-  extends PrintElementBaseOptions,
-    PrintOptions {}
-
-type PrintElementToPdfOptionsDiscriminated = PrintElementToPdfOptions & {
-  printToPdf: true;
-};
-type PrintElementOptionsDiscriminated = PrintElementOptions & {
-  printToPdf: false;
-};
-type PrintElementSuperOptionsDiscriminated =
-  | PrintElementToPdfOptionsDiscriminated
-  | PrintElementOptionsDiscriminated;
-
 // Render an element and print it. The function to render the element takes a
 // callback to indicate when the component has finished rendering and is ready
 // to be printed. This accommodates components that may want to do multiple
 // renders or post-processing before being ready to print.
-async function printElementWhenReadySuper(
+async function printElementWhenReadySuper<PrintResult>(
```

Comment:
Nit: `super` has a very specific connotation for me having to do with classes. I would use `helper` to describe this